docs/articles/clustering/cluster-sharding.md

```diff
@@ -143,6 +147,65 @@ akka.cluster.sharding.journal-plugin-id = <plugin>
 akka.cluster.sharding.snapshot-plugin-id = <plugin>
 ```
 
+> [!IMPORTANT]
```

Comment:
@to11mtm believe I addressed https://github.com/akkadotnet/akka.net/pull/6124#issuecomment-1261316642 here as my resolution to #6123